tests/test_video_downloader.py

```
@@ -0,0 +1,20 @@
+import pytest
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)